metadata/src/main/java/org/apache/kafka/controller/metrics/QuorumControllerMetrics.java

```diff
@@ -76,8 +70,6 @@ public class QuorumControllerMetrics implements AutoCloseable {
     private final AtomicLong dualWriteOffset = new AtomicLong(0);
```

Comment:
Is dual-write metrics still used in production code after `KRaftMigrationDriver` is removed?

Reply:
no